docs/release/release-2.0.rst

```diff
@@ -10,6 +10,7 @@ New Features
 * Change the default configuration directory to ``~/.prestoadmin`` and the
   default log directory to ``~/.prestoadmin/log``.
 * Remove the requirement for running and installing presto-admin with sudo.
```

Comment:
I think we need some sort of explanation though, b/c it's confusing that we say that we "removed the requirement for sudo" when you still do need sudo.